pkg/detectors/rubygems/rubygems.go

```diff
@@ -47,7 +47,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://rubygems.org/api/v1/gems", nil)
```

Comment:
i believe these changes on rubygems are duplicate, and were already merged in on the PR: https://github.com/trufflesecurity/trufflehog/pull/1781

Reply:
Yes, thats correct. They are indeed duplicate

Reply:
could you remove those changes from this PR pls

Reply:
i tried removing these changes, can you please confirm.

If not, then i will close this PR and open another one